tools/sync-diff-inspector.md

```diff
@@ -0,0 +1,214 @@
+---
+title: sync-diff-inspector 用户文档
+category: tools
+---
+
```

Comment:
We should add a first-level heading and put the "title" here. Keep the title consistent with that in metadata.